### ISSUE TITLE
management plugin crash when re-joining existing cluster

### ISSUE THREAD

**Issue by cap10morgan** [aweber/rabbitmq-autocluster-consul#5]
I just tried out the 0.2.0 release and ran into this crash when a node attempts to re-join a cluster: https://gist.github.com/cap10morgan/2b5d402ae271c10245bd

Looks like something is crashing the management plugin.

**Comment by gmr** [comment]
What other plugins are enabled? I've not been able to reproduce this, with 3.5.1, but don't have any reason to think it would be a problem with 3.5.0. Are there any other config settings you can share?

**Comment by cap10morgan** [comment]
I'm using the [rabbitmq:3.5.0-management Docker image](https://registry.hub.docker.com/_/rabbitmq/). Looks like the plugins (besides this one) are:

* rabbitmq_management
* rabbitmq_web_dispatch
* webmachine
* mochiweb
* rabbitmq_management_agent
* amqp_client

**Comment by cap10morgan** [comment]
I'm still seeing this with RabbitMQ 3.5.3. Would it help if I sent you the `erl_crash.dump` file? Or anything else?

**Comment by cap10morgan** [comment]
Oh, the `/etc/rabbitmq/rabbitmq.config` is:

```erlang
[{rabbit, [{loopback_users, []}]}, {rabbitmq_autocluster_consul, [{consul_host, "10.36.136.17"}, {cluster_name, "rabbit@rabbitmq"}]}].
```

**Comment by cap10morgan** [comment]
@gmr When you tried to reproduce this, did you have the rabbitmq_management and rabbitmq_management_agent plugins enabled on all cluster nodes or just one? I have them enabled on all nodes.

**Comment by gmr** [comment]
I finally reproduced this and am working towards a fix that will be in 0.3

**Comment by gmr** [comment]
This should be addressed in 0.3.1. Please let me know if you continue to have issues.

**Comment by patrickhwood** [comment]
I am seeing this with 0.4.1 and RMQ 3.5.3 (not 100% of the time).  To fix it, I have to manually forget_cluster_node on a clustered instance, and kill epmd and clear out ~rabbitmq/mnesia on the crashed instance.  Is there any way to automatically force a forget_cluster_node on a non-responding node?
